### ISSUE TITLE
java.lang.NoClassDefFoundError: io/micronaut/test/extensions/junit5/MicronautJunit5Extension in 1.15.0

### ISSUE THREAD

**Issue by eaxdev** [database-rider/database-rider#217]
I updated to new version: `1.15.0`. I added this in my `build.gradle.kts`:

```kotlin
testImplementation("com.github.database-rider:rider-junit5:1.15.0") 
```

When I try to start my test, I get this error:

```
io/micronaut/test/extensions/junit5/MicronautJunit5Extension
java.lang.NoClassDefFoundError: io/micronaut/test/extensions/junit5/MicronautJunit5Extension
	at com.github.database.rider.junit5.DBUnitExtension.getMicronautApplicationContext(DBUnitExtension.java:270)
	at com.github.database.rider.junit5.DBUnitExtension.getTestConnection(DBUnitExtension.java:125)
	at com.github.database.rider.junit5.DBUnitExtension.executeDataSetForCallback(DBUnitExtension.java:369)
	at com.github.database.rider.junit5.DBUnitExtension.beforeAll(DBUnitExtension.java:330)
```

But, I don't use `Micronaut`, I use `SpringBoot`. How I can disable it?

**Comment by eaxdev** [comment]
I solved it. I added incorrect dependency. It work for me: `testImplementation("com.github.database-rider:rider-spring:1.15.0")`

**Comment by rmpestano** [comment]
Hi @eaxdev, 

I'll reopen because that's indeed the case, we didn't catch that because I just found that SpringBootSample tests are not running.

Thank you

**Comment by rmpestano** [comment]
Hi again, 
It is fixed in `v1.15.1` which should be in maven central in next hours. 

Now you can use JUnit5 and SpringBoot as in our [sample application](https://github.com/database-rider/database-rider/blob/fd7fc1a9582d7df11bf7aa8d3b41cbd89bde63ad/rider-examples/spring-boot-dbunit-sample/src/test/java/com/github/database/rider/springboot/SpringBootDBUnitTest.java#L19-L20).

Thank you for the heads up!
